### ISSUE TITLE
Test videos Apis on Facebook as well

### ISSUE THREAD

**Issue by arn-the-long-beard** [Ringrev/facebook-api-rs#6]
### Overview of feature 📝
Use the library we developed to upload a video to facebook

### Which problem is the feature or enhancement solving❓
Ensure that the api is working as expected and prepare us to any problem that could come.

### Documentation & Help ℹ️
https://developers.facebook.com/docs/video-api

**Comment by osain-az** [comment]
We looked at post  scheduling ( by allowing the different platforms  api to handle it )  but we found out that   Youtube can only  schedule for  a private video therefor  all scheduling post should  should be handled at the frontend instead of the APIs. 


**Facebooking video posting methods **
since facebook have two methods of uploading video (resumable  and non_resumable ), it might be nice to implement three different method for posting which are :  
1. Method that allows positing of video within  1 gb and 2o minutes  ( using non_resumable method ). In this method if you  upload file above error should be generated.
2.  Method for implementing  videos within  4 gb  and 40 minutes  ( resumable method). 
3. Method that takes the video and decide which of the above mentioned methods to use.
